packages/test/test-end-to-end-tests/src/test/gc/gcTrailingOps.spec.ts

```diff
@@ -17,200 +17,230 @@ import {
 	ITestDataObject,
 	TestDataObjectType,
 } from "@fluid-private/test-version-utils";
-import { stringToBuffer } from "@fluid-internal/client-utils";
-import { ISummaryTree } from "@fluidframework/protocol-definitions";
+import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
 import { IGCRuntimeOptions } from "@fluidframework/container-runtime";
-import { getGCStateFromSummary } from "./gcTestSummaryUtils.js";
-import { defaultGCConfig } from "./gcTestConfigs.js";
+import { delay } from "@fluidframework/core-utils";
+import { channelsTreeName, gcTreeKey } from "@fluidframework/runtime-definitions";
+import { getGCDeletedStateFromSummary, getGCStateFromSummary } from "./gcTestSummaryUtils.js";
 
 describeCompat("GC trailing ops tests", "NoCompat", (getTestObjectProvider) => {
```

Comment:
What do you think of adding a big block comment here giving context on these tests?  Covering points like:

- What is a trailing op
- Why are they interesting for GC
- What are the concerns we are trying to cover here
- How are the tests organized

Reply:
Also, should we start putting something like "[LOCAL ONLY]" in the test suite name when we do the skip-if-not-local thing?

Reply:
Good catch. I have tried to add description to every GC test so this makes it consistent with that model.

Reply:
> Also, should we start putting something like "[LOCAL ONLY]" in the test suite name when we do the skip-if-not-local thing?

What does that help with?

Reply:
To make it obvious.  Not a big deal

Reply:
If we want to do this, I would rather do it in the other planned work of making all GC tests run against local.